admin/src/app/Components/hotel/hotel.component.css

```diff
@@ -0,0 +1,13 @@
+mat-card {
+    margin-left: 17%;
+    margin-right: 17%;
+    padding: 2%;
+}
+
+mat-progress-spinner {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    z-index: 5;
```

Comment:
Why `5`?

Reply:
It should be at least 1 for other page elements to do not overlap loading spinner